src/oc_erchef/apps/chef_index/src/chef_solr.erl

```diff
@@ -44,22 +107,32 @@ make_query_from_params(ObjType, QueryString, Start, Rows) ->
     FilterQuery = make_fq_type(ObjType),
     %% 'sort' param is ignored and hardcoded because indexing
     %% scheme doesn't support sorting since there is only one field.
-    Sort = "X_CHEF_id_CHEF_X asc",
+    Sort = binary_to_list(id_field()) ++ " asc",
```

Comment:
@marcparadise I believe that comment refers to the user-supplied sort parameter.  The Sort parameter we send to Solr and Cloudsearch are definitely not ignored and ensure that we can page results consistently.